### PR TITLE
Fix for the ping command

### LINKED ISSUE #42: Output of ping double of actual ping
Because the signal travels back to the bot

### PR DESCRIPTION
**Pull request by NilEis**
fixes #42

### CODE REVIEW

File: src/main/java/ml/codeboy/thebot/commands/sound/Join.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    public Join() {
+        super("join");
+    }
+
+    @Override
+    public void run(CommandEvent event) {
+        if(ensureConnected(event)) {
+            event.reply(Util.sign(event.getBuilder().setTitle("Hello"), event));
+            return;
+        }
```
Comment (the-codeboy) [inline src/main/java/ml/codeboy/thebot/commands/sound/Join.java:16]: There should be some kind of answer when ensureconnected returns false

**Review by the-codeboy — CHANGES_REQUESTED** [review]
There's a couple of issues I adressed

**Comment by the-codeboy** [comment]
That wasn't really how I meant it but ok...

**Comment by NilEis** [comment]
I mdae a mistake with the branch :/

**Pull request closed without merging**